### PR TITLE
fix: improve app's bootstrap HTML metadata

### PR DESCRIPTION
**Pull request by jphastings**
Some simple changes to remove defaults from the CreateReactApp origins:

- Improves the HTML title to more specifically match this app (vs. other IPFS produced sites)
- Adds a meta description (pulled from the Github repo)
- Adds an IPFS theme colour in HTML meta
- Adds canonical link to bootstrap HTML

### CODE REVIEW

**Review by SgtPooki — APPROVED** [review]

**Comment by SgtPooki** [comment]
Thanks for this @jphastings, unfortunately, our pinning CI is currently broken due to Estuary sunsetting their pinning service and our ipfs-cluster intermittently failing on pinning this content. I am tracking some work to address this in https://github.com/ipfs/ipfs-webui/issues/2161

**Comment by jphastings** [comment]
Thanks for keeping me up to date, let me know if there's anything you'd like me to do!

File: public/index.html [hunk 1]
```diff
@@ -3,9 +3,10 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="theme-color" content="#000000">
+    <meta name="theme-color" content="#55B8BE">
```
Comment (SgtPooki) [inline public/index.html:6]: to better align with navbar color of webui: https://github.com/ipfs-shipyard/ipfs-css/blob/main/README.md#colors

```suggestion
    <meta name="theme-color" content="#0b3a53">
```

**Review by SgtPooki — APPROVED** [review]

File: public/index.html [hunk 2]
```diff
@@ -21,22 +22,23 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>IPFS</title>
+    <title>WebUI | IPFS</title>
+    <link rel="canonical" href="https://webui.ipfs.io/">
```
Comment (lidel) [inline public/index.html:26]: cc #2223 so we don't forget to update this

**Review by lidel — APPROVED** [review]
Lgtm, small suggestions inline but also fine to disregard.

**Pull request merged**